.gitignore

```diff
@@ -8,4 +8,4 @@ docs/src
 *.tmp
 *.swo
 *.swp
-/private/
+/private/
```

Comment:
Sorry for the super minor nitpick, but let's keep the newline at the end of the file, so we don't have this in the diff

Reply:
No problem, my bad for missing that, nice catch.